compiler/docgen.nim

```diff
@@ -1499,7 +1499,9 @@ proc genOutFile(d: PDoc, groupedToc = false): string =
   # Extract the title. Non API modules generate an entry in the index table.
   if d.meta[metaTitle].len != 0:
     title = d.meta[metaTitle]
-    let external = presentationPath(d.conf, AbsoluteFile d.filename).changeFileExt(HtmlExt).string.nativeToUnixPath
+    let external = d.destFile.AbsoluteFile
+                    .relativeTo(d.conf.outDir, '/')
+                    .changeFileExt(HtmlExt).string
```

Comment:
Hm, if the chained-call style is going to be used, this might be better:
```suggestion
    let external = AbsoluteFile(d.destFile)
      .relativeTo(d.conf.outDir, '/')
      .changeFileExt(HtmlExt)
      .string
```
this way, it's a bit more clear that AbsoluteFile is a type initializer.